addons/project/models/project_task.py

```diff
@@ -794,7 +794,7 @@ def _ensure_fields_write(self, vals, check_group_user=True, defaults=False):
 
         for fname, value in vals.items():
             field = self._fields[fname]
-            if field.type == 'many2one':
```

Comment:
Not sure I understand this change. Shouldn't we rather move the field from WRITABLE_FIELDS to READABLE_FIELDS?

Reply:
Right, but the field is intended to be readonly in the portal now, so what is it doing it WRITABLE_FIELDS?